searchcore/src/vespa/searchcore/fdispatch/search/nodemanager.h

```diff
@@ -48,9 +48,9 @@ class FastS_NodeManager : public config::IFetcherCallback<PartitionsConfig>
 
     volatile bool     _checkTempFailScheduled;
     volatile bool     _shutdown;
+    volatile uint32_t _waitUpMS;
```

Comment:
All these should probably be atomics

Reply:
They should, but as the life of fdispatch is counted in weeks I just went with what broke the least with the existing style.